src/index.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+const tempDisplay = document.getElementById('temp-display');
+const landscapeImg = document.getElementById('landscape-img');
+const cityName = document.getElementById('city-name-display');
+const cityInputBox = document.getElementById('city-name');
+
+const state = {
+  temp: 80,
+  city: cityName.textContent,
+  color: 'rgb(248, 163, 163)',
+};
+
+// Set default values
+tempDisplay.textContent = `${state.temp}°`;
+tempDisplay.style.color = state.color;
+
+const updateTempDisplay = (x) => {
+  state.temp += x;
+  tempDisplay.textContent = `${state.temp}°`;
+
+  if (state.temp > 79) {
+    tempDisplay.style.color = 'rgb(248, 163, 163)';
```

Comment:
By setting the `style` attribute directly on an element, we’re doing the javascript equivalent of adding inline CSS styles, which is not considered best practice. The preference would be to create CSS rules for the styles, likely attached to a class name, then set the element's `className` attribute to the appropriate value for the style we want to attach.

So in our CSS we'd have rule sets:
```CSS
...

.red {
    color: red;
}

.orange {
    color: orange;
}

... /* (and so on for the other colors necessary) */
```

And here we'd do something like:
```javascript
...
if (state.temp >= 80) {
    currentTemp.className = 'red';
...
```